NetCoreRestApi/DataLayer/Entities/BaseEntity.cs

```diff
@@ -3,6 +3,8 @@ namespace DataLayer.Entities
 {
     public class BaseEntity : IEntity
     {
-        public int Id { get; set; }        
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
```

Comment:
Create ItemEntity with Name and Description

Reply:
done